File: 1008.cs
using System;
class URI {
    static void Main(string[] args) { 
        int numfunc = int.Parse(Console.ReadLine());
        int hrtrab = int.Parse(Console.ReadLine()); 
        float valorhr = float.Parse(Console.ReadLine());
        float salario = hrtrab * valorhr;
        Console.WriteLine($"NUMBER = {numfunc}");
        Console.WriteLine($"SALARY = U$ {salario:F2}");

    }

}

File: 1008.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main(){
    int numfunc, hrtrab;
    double valorhr, salario;
    cin >> numfunc;
    cin >> hrtrab;
    cin >> valorhr;
    salario = hrtrab * valorhr;
    cout << "NUMBER = " << numfunc << "\n";
    cout << fixed << setprecision(2);
    cout << "SALARY = U$ " << salario << endl;
}
